packages/components/src/components/Table/Table.js

```diff
@@ -117,19 +117,19 @@ const Table = props => {
     emptyTextAllNamespaces,
     emptyTextSelectedNamespace,
     filters,
+    hasDetailRows,
```

Comment:
You had asked about the naming convetion here. Only other thought was `hasDetails`

Alternatively scrub the extra `hasDetailRows` prop and just set the `size` prop to `xl` for the Table component in PipelineRuns and TaskRuns? I dont recall from the walkthrough if you had a case in mind that meant the extra prop was needed

Reply:
I prefer `hasDetails`, I'll update it. Thanks.

Yes it's needed to control positioning of the batch action checkboxes + overflow menu icons as they're not vertically centered in this case as they would be by just setting `size="xl"`, as well as some other tweaks to spacing of the content (see the `tkn--table-with-detail-rows` class which I'll also rename)

Reply:
Updated 👍